editor/src/clj/editor/scene_tools.clj

```diff
@@ -108,7 +108,8 @@
     :scale-xy [0.0 0.0 1.0]
     :scale-xz [0.0 1.0 0.0]
     :scale-yz [1.0 0.0 0.0]
-    :scale-uniform [0.0 0.0 1.0]))
+    :scale-uniform [0.0 0.0 1.0]
+    [0.0 0.0 1.0]))
```

Comment:
Do we really want this default case? It's kind of nice that the `case` expression is exhaustive, and I'm not sure there is a sensible default behavior for this without context? 🤔

Reply:
I wanted to avoid adding a key from a different ns here, but since we already pass `:move-pivot` to `gen-manip-renderable`, we might as well add an explicit case.